ahnlich/ai/src/cli/server.rs

```diff
@@ -92,6 +107,11 @@ pub struct AIProxyConfig {
     /// List of ai models to support in your aiproxy stores
     #[arg(long, required(true), value_delimiter = ',')]
     pub(crate) supported_models: Vec<SupportedModels>,
+
+    /// Directory path for storing the model artifacts
+    #[arg(long, default_value_os_t =
+    DEFAULT_CONFIG.get_or_init(AIProxyConfig::default).model_cache_location.clone())]
```

Comment:
I like this trick... could you please do it for other defaults? so we reduce our duplication

Reply:
Makes sense. I'll do that.